packages/log-error/README.md

```diff
@@ -199,6 +200,28 @@ logRecoverableError({
 > **Note**
 > There's no need to include the `x-request-id` header in this array, as this is automatically included as `request.id` in the logs.
 
+#### `options.logger`
```

Comment:
Is it worth including the scenarios in this PR's description? e.g. why it might be necessary to use a custom logger (for masking or including specific base data).

Reply:
Done 👍

Reply:
I phrased that really badly - I meant to ask whether the examples included in the PR description (of the scenarios when it might necessary to use a custom logger) could be included in this README.

Reply:
Ooh! 😂 my bad. Yes probably, will do it as a separate PR